src/polymer/polymer-element.ts

```diff
@@ -221,14 +221,15 @@ function _getFlattenedAndResolvedBehaviors(
     //     array, which should be the most recently defined one.
     const foundBehavior = document.getOnlyAtId('behavior', behavior.name);
     if (!foundBehavior) {
-      throw new WarningCarryingException({
+      document.warnings.push({
```

Comment:
Or we could filter for this at `getWarnings` maybe, but we'd have a problem with duplicate warnings I think.